### DISCUSSION TITLE
直播间红包提醒功能

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#4369] [功能建议 / Ideas]
![QQ截图20230811201921](https://github.com/the1812/Bilibili-Evolved/assets/14146754/799fb4fd-c762-45c6-abb0-6a2503d9ec13)

自从礼物栏改版后，网页端红包需要点开箭头才能看到

所以能不能移出外面防止漏掉呢，如果能像天选时刻有弹窗提醒就更好了

**Comment by CrazyboyQCD** [comment]
建议直接使用https://greasyfork.org/zh-CN/scripts/439169-b%E7%AB%99%E7%9B%B4%E6%92%AD%E8%87%AA%E5%8A%A8%E6%8A%A2%E7%BA%A2%E5%8C%85

**Reply by u2shana** [reply]
这个没有单独的提醒功能，而且有人反馈这样抢红包会黑号
